### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by katrinleinweber**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

This is a follow-up to a bunch of recent, content-adding PRs. I'm not sure how best to ensure detection of each such changes. I scrolled through the [commit list](https://github.com/github/opensource.guide/commits/master) and looked at merge commits that seemed to add content.

---

Dear @jens1o, do you want to review again, like in #577? The English originals are linked as hashes in the commit descriptions.

Related to #628

### CODE REVIEW

File: _articles/de/best-practices.md [hunk 1]
```diff
@@ -179,6 +179,8 @@ Sie müssen nicht alles selbst machen. Die Gemeinschaft Ihres Projekts existiert
 
 Wenn Sie auf der Suche nach Mitwirkenden sind, fragen Sie doch einfach mal herum.
 
+Sie können auch [Issues, die für Anfänger einfach genug sind, explizit markieren](https://help.github.com/en/articles/helping-new-contributors-find-your-project-with-labels). GitHub weist auf solche Issues vermehrt hin, sodass beitragswillige Neulinge zu nützlicher Mitarbeit ermuntert werden.
```
Comment (jens1o) [inline _articles/de/best-practices.md]: As usual, do we want to put `(Englisch)` here for links that aren't (yet) accessible in German?

**Comment by jens1o** [comment]
Of course I do. :)

**Comment by katrinleinweber** [comment]
Cool, thank you :-)

**Comment by SAVAGE6329** [comment]
> * [x]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
> * [x]  Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
> 
> This is a follow-up to a bunch of recent, content-adding PRs. I'm not sure how best to ensure detection of each such changes. I scrolled through the [commit list](https://github.com/github/opensource.guide/commits/master) and looked at merge commits that seemed to add content.
> 
> Dear @jens1o, do you want to review again, like in #577? The English originals are linked as hashes in the commit descriptions.
> 
> Related to #628

**Comment by mmoo301** [comment]
#1118

**Review by saintyvesfsya — COMMENTED** [review]
Ok

**Review by saintyvesfsya — COMMENTED** [review]
Goo

**Review by DmitrijKaravaea — COMMENTED** [review]
na @katrinleinweber

**Pull request merged**